src/pages/countries.js

```diff
@@ -0,0 +1,82 @@
+import { useQuery, gql } from "@apollo/client";
+import { Link, useLocation, useHistory } from "react-router-dom";
+import CountriesList from "../components/CountriesList";
+import { useState, useEffect } from "react";
+
+export const COUNTRIES_QUERY = gql`
+  query allCountries($limit: Int, $skip: Int) {
+    countries(limit: $limit, skip: $skip) {
+      id
+      name
+      vatRate
+      languages {
+        id
+        name
+      }
+    }
+  }
+`;
+
+function useQueryUrl() {
+  return new URLSearchParams(useLocation().search);
+}
```

Comment:
In real scenario it can be more convenient, for example above function may return object with params that you can later destructure, so the usage might be `const { page } = useQueryUrl()` but this is also cool